apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -94,6 +97,14 @@ export function InstructorQuestionSettings({
             course: resLocals.course,
             urlPrefix: resLocals.urlPrefix,
           })}
+          <link
+            href="${nodeModulesAssetPath(
+              resLocals.use_bootstrap_4
+                ? 'tom-select/dist/css/tom-select.bootstrap4.css'
+                : 'tom-select/dist/css/tom-select.bootstrap5.css',
+            )}"
+            rel="stylesheet"
+          />
```

Comment:
This should go in `<head>`.

Reply:
Moved this to the `<head>` in [d6653ad](https://github.com/PrairieLearn/PrairieLearn/pull/10868/commits/d6653ad6230c4eb5a5c64ad81b51764e592ada17).